### ISSUE TITLE
BridgeOS 6 appears to be patched to prevent checkra1n jailbreak

### ISSUE THREAD

**Issue by apocnz** [checkra1n/BugTracker#2208]
**Tell us about your setup:**
1. What iDevice are you using?

A1932 / Macbook Air 2018

2. On what version of iOS is it?

Big Sur

3. What version of checkra1n are you using?

12.4

4. What is your host system (OS version? Hackintosh? VM? etc.)?

MBP 2010 running Catalina + tried Hackintosh running Big Sur

5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?

USB A - > USB C on MBP 2010
USB C -> USB C on Hackintosh

**What are the steps to reproduce the issue?**

1. Tried Checkra1n cli to jailbreak T2 
2. Tried versions 11, 12.2 and 12.4

**What do you expect, and what is happening instead?**

Usual Jailbreak with SSH access.

Jailbreak fails, appears BridgeOS 6.6 has been patched to prevent usual jailbreak process

**Any other info, error logs, screenshots, ...?**

After attempted jailbreak, T2 appears to stop entering DFU normally

Just submitting this in case its not known yet, but it appears the last two IPSW's for BridgeOS 2,1
have been patched to prevent the checkra1n jailbreak from succeeding.  Tried restoring last signed IPSW
for BridgeOS (Oct 25 release), no go.

**Comment by genknife** [comment]
same problem

**Comment by wasback** [comment]
Yes, I hope checkra1n team will have time to bring back T2 jailbreak on new bridgeos ;)
Same on a A1989, bridgeOS 6.2...

Please do fix it with all of our support !

**Comment by Pavel9922** [comment]
Same problem 😭

**Comment by Tirante-el-Blanco** [comment]
same
